### DISCUSSION TITLE
dbt command to list nodes selected by `--selector`, `--select`, `--model`, `--exclude` ?

### DISCUSSION THREAD

**Discussion by p-hebert** [dbt-labs/dbt-core#5431] [Q&A]
Hi!

Related to #4456 

I am writing a custom Airflow operator around `dbt snapshot` to run the command incrementally over multiple snapshots of a table over time, and I said operator is designed in a way that it is made to target a single model at a time.
I am looking for a way to assert the expectation that the selector passed only concerns a single model.

Is there a way to list all the nodes selected by the selection flags of a certain command, without running any action?
A bit like an early/partial dry-run mode?

Thanks!

**Comment by tasdoggy** [comment]
dbt ls -s xxxx

**Comment by graciegoheen** [comment]
Yes! This is `dbt list` command :)
